colpali_engine/utils/dataset_transformation.py

```diff
@@ -82,31 +82,52 @@ def load_train_set_with_tabfquad() -> DatasetDict:
     return ds_dict
 
 
-def load_docmatix_ir_negs() -> Tuple[DatasetDict, Dataset]:
+def load_docmatix_ir_negs() -> Tuple[DatasetDict, Dataset, str]:
     base_path = "./data_dir/" if USE_LOCAL_DATASET else "Tevatron/"
     dataset = cast(Dataset, load_dataset(base_path + "docmatix-ir", split="train"))
-    dataset = dataset.select(range(100500))
+    # dataset = dataset.select(range(100500))
 
     dataset_eval = dataset.select(range(500))
     dataset = dataset.select(range(500, len(dataset)))
     ds_dict = DatasetDict({"train": dataset, "test": dataset_eval})
 
     base_path = "./data_dir/" if USE_LOCAL_DATASET else "HuggingFaceM4/"
-    anchor_ds = cast(Dataset, load_dataset(base_path + "Docmatix", split="train"))
+    anchor_ds = cast(Dataset, load_dataset(base_path + "Docmatix", "images", split="train"))
 
-    return ds_dict, anchor_ds
+    return ds_dict, anchor_ds, "docmatix"
+
+def load_wikiss() -> Tuple[DatasetDict, Dataset, str]:
+    base_path = "./data_dir/" if USE_LOCAL_DATASET else "Tevatron/"
+    dataset = cast(Dataset, load_dataset(base_path + "wiki-ss-nq", data_files="train.jsonl", split="train"))
+    # dataset = dataset.select(range(400500))
+    dataset_eval = dataset.select(range(500))
+    dataset = dataset.select(range(500, len(dataset)))
+    ds_dict = DatasetDict({"train": dataset, "test": dataset_eval})
+
+    base_path = "./data_dir/" if USE_LOCAL_DATASET else "HuggingFaceM4/"
+    anchor_ds = cast(Dataset, load_dataset(base_path + "wiki-ss-corpus", split="train"))
+
+    return ds_dict, anchor_ds, "wikiss"
 
 
 def load_train_set_ir_negs() -> Tuple[DatasetDict, Dataset]:
```

Comment:
Haven't you missed a str in the output typing?

```suggestion
def load_train_set_ir_negs() -> Tuple[DatasetDict, Dataset, str]:
```